### PR TITLE
Issue 1901: Copy our pull request template to the rest of the Strongbox projects

### PR DESCRIPTION
**Pull request by pawelpaszki**
Fixes strongbox/strongbox#1901.

### CODE REVIEW

**Comment by carlspring** [comment]
Hi,

Thanks for offering to help, but this is incorrect.

We already have a template itn the Strongbox project. This was mentioned in the issue. That template needs to be copied over. I have no idea where you got this template from, but this is not the correct one.

Also, could you please:
* Use a title for the pull request that matches the action issue from our issue tracker
* Link back to the actual task

(I've done both of the above for you for this pull request, but please go over the rest of your pull requests and fix it. Thanks!).

Martin

**Comment by pawelpaszki** [comment]
Should be fixed now

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Pull Request Description
+
+This pull request closes strongbox/strongbox#
+
+# Acceptance Test
+
+* [ ] Building the code with `mvn clean install -Dintegration.tests` still works.
+* [ ] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
+* [ ] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
+* [ ] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.
```
Comment (carlspring) [inline .github/PULL_REQUEST_TEMPLATE.md:10]:
```suggestion
* [ ] Building the code of the `strongbox-parent` with `mvn clean install` still works.
* [ ] Building the code of the `strongbox` project with `mvn clean install -Dintegration.tests` still works.
* [ ] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [ ] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [ ] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+This pull request closes strongbox/strongbox#
+
+# Acceptance Test
+
+* [ ] Building the code of the `strongbox-parent` with `mvn clean install` still works.
+* [ ] Building the code of the `strongbox` project with `mvn clean install -Dintegration.tests` still works.
+* [ ] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
+* [ ] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
+* [ ] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.
```
Comment (carlspring) [inline .github/PULL_REQUEST_TEMPLATE.md:11]:
```suggestion
* [ ] Building the code of the [`strongbox-parent`](https://github.com/strongbox/strongbox-parent) with `mvn clean install` still works.
* [ ] Building the code of the [`strongbox`](https://github.com/strongbox/strongbox) project with `mvn clean install -Dintegration.tests` still works.
* [ ] Running `mvn spring-boot:run` in the `strongbox-web-core` still starts up the application correctly.
* [ ] Building the code and running the `strongbox-distribution` from a `zip` or `tar.gz` still works.
* [ ] The tests in the [`strongbox-web-integration-tests`](https://github.com/strongbox/strongbox-web-integration-tests/) still run properly.
```

**Comment by carlspring** [comment]
Thank you for you contribution! :)

**Pull request merged**